docsrc/conf.py

```diff
@@ -50,12 +60,18 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ['_build', '_tmp', 'Thumbs.db', '.DS_Store']
 
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = True
+
 # -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_material'
+html_theme = 'pytorch_sphinx_theme'
```

Comment:
Is this for addressing vulnerability reports?

Reply:
This is the new PyTorch theme to match the rest of the documentation